doc/dev/pylint_checking.md

```diff
@@ -0,0 +1,71 @@
+# Python SDK Pylint Guide
+
+Cheat sheet for pylint general guidelines in the Python client SDK library. 
+
+# Table of contents
+  - [How to Prepare your SDK?](#how-to-prepare-your-sdk-for-a-new-pylint-update)
+  - [General Guidance](#general-guidance)
+    - [What is Pylint?](#what-is-pylint)
+    - [How to run Pylint?](#how-to-run-pylint)
+  - [Ignoring Pylint Checkers](#ignoring-pylint-checkers)
+  - [Pylint Warnings and Where to Find Them](#pylint-warnings-and-where-to-find-them)
+    - [Next Pylint](#next-pylint)
+
+# How to prepare your SDK for a new pylint update?
+
+Check each SDK's `Run Pylint Next` output in the [test-weekly CI pipeline](https://dev.azure.com/azure-sdk/internal/_build?pipelineNameFilter=python%20*%20tests-weekly). If there is no corresponding test-weekly pipeline, run `next-pylint` locally with `tox` as described in [How to run Pylint?](#how-to-run-pylint). In order to ensure that the SDK pipeline will not break when pylint is updated, make sure to address all pylint warnings present. 
+
+See below for more information on pylint checkers and how to ignore warnings if needed.
+
+# General Guidance 
+
+### What is pylint?
+
+Pylint is a set of (mostly) [astroid](https://pylint.pycqa.org/projects/astroid/en/latest/index.html) based checkers that run static analysis on your code and check that your formatting aligns with Python's style guide (i.e [PEPs](https://peps.python.org/)). 
+
+In the Azure SDK for Python repository, in addition to the standard pylint library, we added custom checkers that help to align our libraries with the [Azure SDK for Python Guidelines](https://guidelinescollab.github.io/azure-sdk/python_introduction.html).
+
+### How to run Pylint?
+
+One way to run pylint is to run at the package level with tox:
+
+    .../azure-sdk-for-python/sdk/eventgrid/azure-eventgrid>tox -e pylint -c ../../../eng/tox/tox.ini
+
+If you don't want to use tox you can also install and run pylint on its own:
+
+ - If taking this approach, in order to run with the pylintrc formatting and the custom pylint checkers you must also install the custom checkers and `SET` the pylintrc path. 
+
+- More info on how to run the custom pylint checkers [here](https://github.com/Azure/azure-sdk-tools/blob/main/tools/pylint-extensions/pylint-guidelines-checker/README.md)
+
+        pip install pylint
```

Comment:
did you mean to have the commands here in addition to the link? if not, maybe you can just pick one. or if you did mean to add a link for additional information, maybe you can move the link to a link under the commands and say something like `Additional information about custom checkers [here]`.